### PR TITLE
Adding working draft link to README for better contrib visibility

### PR DESCRIPTION
**Pull request by matthewflannery**
Adding the current WIP documentation so that it is easier to find for contributors rather than referring to previously closed issues. cc @anvega

### CODE REVIEW

**Review by brandtkeller — COMMENTED** [review]
Great call! This will make discovery (and also some general ease of use) much better.

**Comment by anvega** [comment]
@matthewflannery Thank you for the addition! Could you please rebase your branch and sign the commit to ensure the DCO check passes?

File: community/automated-governance/README.md [hunk 1]
```diff
@@ -17,6 +17,10 @@ The scope of this project includes:
 - Creation of best practice guidelines and documentation.
 - Potential development of tooling or integration patterns for common CI/CD platforms.
 
+## WIP Documentation
```
Comment (brandtkeller) [inline community/automated-governance/README.md:20]:
```suggestion
## Draft
```

**Comment by matthewflannery** [comment]
I've rebased this now. 

Checks are passing except the spelling linter: 
community/working-groups/automated-governance/README.md:31:13 - Unknown word (Andrés) Suggestions: [andres, Andres, andros, Andros, Andes] @anvega Might need to change your name legally mate :P

**Review by brandtkeller — APPROVED** [review]
Approving so that we can continue to iterate forward on what content to expose here as a group.

**Pull request merged**